roles/openshift_health_checker/openshift_checks/logging/podlogs.py

```diff
@@ -0,0 +1,135 @@
+"""
+Module for ensuring logs from pods can be queried in a reasonable amount of time.
+"""
+
+import json
+import time
+
+from uuid import uuid4
+
+from openshift_checks import get_var, OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+
+
+ES_CMD_TIMEOUT_SECONDS = 30
+ES_DEFAULT_HOST = "logging-es"
+ES_DEFAULT_INDEX = "project.logging*"
+
+
+class Podlogs(LoggingCheck):
+    """Check that logs from ElasticSearch pods can be queried within a reasonable amount of time."""
+
+    name = "podlogs"
```

Comment:
Maybe pod_logs (separate words with underscore as we do for other check names)? Or even something more descriptive/verbose.

Then make the class name a CamelCase version of the name.